CMakeLists.txt

```diff
@@ -135,6 +137,11 @@ elseif(MINGW)
     # redirect to wmain
     # FIXME: cmake 3.13 added target_link_options
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -municode")
+    if(CRASHPAD_WER_ENABLED)
+        message(STATUS "WER support enabled")
+    else()
+        message(STATUS "WER support disabled. Define CRASHPAD_WER_ENABLED = TRUE to enable.")
+    endif()
```

Comment:
Yep, not only is LLVM-mingw ramping up CFG support, but crashpad seems to use the WER handler to accept both:
```
0xC0000602,  // STATUS_FAIL_FAST_EXCEPTION
0xC0000409,  // STATUS_STACK_BUFFER_OVERRUN
```
And AFAIK other things can also lead to fast-fail type exceptions. (Reference?: https://devblogs.microsoft.com/oldnewthing/20190108-00/?p=100655)

The above change to the CMake file only outputs a config log entry as to whether it’s enabled or not - the embedding project still has to decide where to define CRASHPAD_WER_ENABLED to enable it for mingw builds (currently).

Reply:
Yeah, I understand that there is a range of `fastfail` scenarios. We included both `WER` cases in our example and thus integration tests, so when integrating into the CI you will have to disable them. For me, the question was how far the integration of `fastfail` support goes in MinGW since even if you exclude the CFG, things like stack-guarding are implemented differently on GCC (and probably won't lead to `fastfail`, but I am not sure).

In any case, I am OK with enabling WER independent of the support in MinGW. We can also leave this to the developer whether they want it enabled. Right now, that option is not open to the MSVC side, where you need to _target_ a supported platform to enable it. But we might change that.